torch/csrc/Module.cpp

```diff
@@ -628,6 +645,8 @@ static PyMethodDef TorchMethods[] = {
   {"_is_xnnpack_enabled", THPModule_isEnabledXNNPACK, METH_NOARGS, nullptr},
   {"_is_torch_function_enabled", THPModule_isEnabledTorchFunction, METH_NOARGS, nullptr},
   {"_disabled_torch_function_impl", THPModule_disable_torch_function, METH_VARARGS, nullptr},
+  {"_is_tensor", THPModule_is_tensor, METH_O, nullptr},
```

Comment:
Can we document that this is an exact `Tensor`, perhaps by changing the name to not confuse people into thinking it's checking for instance as well?

Reply:
Agreed, the underscore makes it better, but people are still going to misunderstand. `_is_tensor_exact` perhaps